iati/core/exceptions.py

```diff
@@ -28,4 +28,8 @@ class ValidationError(ValueError):
         This is too general to identify many specific problems.
     """
 
-    pass
+    def __init__(self, error_log):
+        """Initialise a ValidationError."""
```

Comment:
Is the thinking that we'll have other types of exceptions? `ValidationError` seems quite general, as noted in the `Warning` docstring.